ui/components/requirements/requirement.jsx

```diff
@@ -15,13 +16,37 @@ export default function Requirement({ requirement }) {
       <div className="req-text col col-10">
         { reqTexts }
         <div className="clearfix mt3">
-          <div className="applies-to mr2">
+          <div className="applies-to mr2 metadata">
             Applies to: [not implemented]
           </div>
-          <div className="sunset-date">
-            Sunset date by { requirement.policy.sunset || 'none' }
+          { requirement.issuing_body ? (
+            <div className="issuing-body metadata">
+              Issuing body: { requirement.issuing_body }
+            </div>
+          ) : (
+            ''
+          ) }
```

Comment:
@tadhg-ohiggins @cmc333333 I'm wondering if we could use React's conditional rendering to make this a little cleaner? https://facebook.github.io/react/docs/conditional-rendering.html
It may be overkill, but maybe we have components for each piece of metadata that render conditionally based on the presence of the value we're checking in these statements.